components/CocktailsOverview/index.js

```diff
@@ -24,7 +24,7 @@ export default function CocktailsOverview({
   return (
     <>
       <StyledIntro>
-        <p>Get inspired with our most popular cocktails: </p>
+        <h2>Get inspired with our most popular cocktails: </h2>
```

Comment:
Instead of changing the html Element, you could use your StyledIntro or make another styled component.